css/header.css

```diff
@@ -20,13 +22,16 @@
 }
 
 /* Header icons get higher opacity for visibility */
-#appmenu li a {
-    opacity : .8;
+#appmenu li a,
+.searchbox input[type="search"],
+#header .header-right > div > .menutoggle,
+#header .header-right > form > .menutoggle {
+	opacity: .9;
```

Comment:
indentation looks odd compared to the rest :wink:

Reply:
Good catch – will fix in a next pull request which touches other stuff too. :)